tests/omhttp-basic.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# This file is part of the rsyslog project, released under ASL 2.0
+
+#  Starting actual testbench
+. ${srcdir:=.}/diag.sh init
+
+port="$(get_free_port)"
+omhttp_start_server $port
+
+error_file=$(pwd)/$RSYSLOG_DYNNAME.omhttp.error.log
```

Comment:
```suggestion
error_file=$RSYSLOG_DYNNAME.omhttp.error.log
```
Leave the pwd part out - $RSYSLOG_DYNNAME is your sole prefix. Else things may break (now in some envs or later after a plumbing change).